documentation/ingestion_server/guides/deploy.md

```diff
@@ -0,0 +1,14 @@
+# Deployment
+
+This codebase is deployed as a Docker image to the GitHub Container Registry
+[ghcr.io](https://ghcr.io). The deployed image is then pulled in the production
+environment. See the [`ci_cd.yml`](../.github/workflows/ci_cd.yml) workflow for
```

Comment:
This link to the `ci_cd.yml` workflow and the other link to the `docker-compose.yml` file below need to be changed to GitHub links. You'll see the following warnings (which will fail the CI docs build) locally due to these:

```
/home/sara/projects/openverse/documentation/ingestion_server/guides/deploy.md:3: WARNING: 'myst' cross-reference target not found: '../.github/workflows/ci_cd.yml' [myst.xref_missing]
/home/sara/projects/openverse/documentation/ingestion_server/guides/deploy.md:8: WARNING: 'myst' cross-reference target not found: 'docker-compose.yml' [myst.xref_missing]
generating indices... genindex done
```

Additionally, the links simply won't work. Clicking this one tries to take you to `/ingestion_server/guides/deploy.html#../.github/workflows/ci_cd.yml`.

For this particular bit of documentation about the GHCR workflow, it's probably better to link to the documentation page for our general image publishing process. In that case, change this one to `/meta/ci_cd/jobs/docker.html#publish-images`